src/dotnet/ShellShim/ShellShimMaker.cs

```diff
@@ -51,17 +60,42 @@ public void EnsureCommandNameUniqueness(string shellCommandName)
             }
         }
 
+        internal void CreateConfigFile(string outputPath, string entryPoint, string runner)
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                throw new PlatformNotSupportedException("Creating a .NET Framework app config is not supported on this platform");
+            }
+
+            if (string.IsNullOrWhiteSpace(entryPoint) 
```

Comment:
Also worth checking if this isn't being validated elsewhere already.

Reply:
I did a quick skim and didn't see in, but @wli3 would know better. I didn't see any tests for this kind of malformed manifest.

Reply:
checked here https://github.com/dotnet/cli/blob/master/src/dotnet/ToolPackage/ToolConfiguration.cs#L21

Reply:
Updated. I removed unneeded checks.